.github/workflows/build.yml

```diff
@@ -148,17 +186,7 @@ jobs:
     environment: ${{ inputs.use_kms && 'oidc_aws_kms_secureboot_signing' || '' }}
     strategy:
       fail-fast: false
-      matrix:
-        arch: [ amd64, arm64 ]
-        target: [ kvm, "kvm_trustedboot", "kvm_trustedboot_tpm2", metal, "metal_trustedboot", "metal_trustedboot_tpm2", gcp, "gcp_trustedboot", "gcp_trustedboot_tpm2", gdch, aws, "aws_trustedboot", "aws_trustedboot_tpm2", azure, ali, openstack, openstackbaremetal, vmware, "metal_pxe", firecracker, "metal-vhost" ]
-        modifier: [ "${{ inputs.default_modifier }}" ]
```

Comment:
not sure completely dropping the `default_modifier` is safe, are we sure we never have a need to overload this?
See comment below on maybe changing the `flavors.yaml` format slightly, then this input could be used to replace / overwrite some default mod in there 🤔

Reply:
@nkraetzschmar Any update on this from your side?
I removed the `modifier` from `flavors.yaml` completely for now and use the modifier values as the features that they represent.

e.g.
```
  59   │       - features:
  60   │           - gardener
  61   │           - _prod
  62   │           - _trustedboot
  63   │           - _tpm2
  ```